cirq-core/cirq/transformers/analytical_decompositions/two_qubit_to_cz_test.py

```diff
@@ -265,4 +265,4 @@ def test_decompose_to_diagonal_and_circuit(v):
     assert cirq.is_diagonal(diagonal)
     combined_circuit = cirq.Circuit(cirq.MatrixGate(diagonal)(b, c), ops)
     circuit_unitary = combined_circuit.unitary(qubits_that_should_be_present=[b, c])
-    cirq.testing.assert_allclose_up_to_global_phase(circuit_unitary, v, atol=2e-6)
+    cirq.testing.assert_allclose_up_to_global_phase(circuit_unitary, v, atol=1e-5)
```

Comment:
[No change needed] I've seen _relative_ difference in excess of `1e-5`, but absolute is still below that mark.
```
Max absolute difference: 5.1897987e-06
Max relative difference: 1.1158966e-05
```

Reply:
yeah I am giving it some room

Reply:
Just hit this again with worse error:
```
Max absolute difference: 4.5085162e-05
Max relative difference: 0.00063814
```